src/Cardano/Wallet/Binary.hs

```diff
@@ -79,9 +79,12 @@ import qualified Data.Set as Set
 
 decodeAddress :: CBOR.Decoder s Address
 decodeAddress = do
-    _ <- CBOR.decodeListLenCanonicalOf 2 -- CRC Protection Wrapper
-    tag <- CBOR.decodeTag -- Mysterious hard-coded tag cardano-sl seems to so much like
-    bytes <- CBOR.decodeBytes -- Addr Root + Attributes + Type
+    _ <- CBOR.decodeListLenCanonicalOf 2
+        -- ^ CRC Protection Wrapper
```

Comment:
The `^` is invalid here, this isn't a valid Haddock syntax :/

Reply:
You're right! I've removed the `^`s.